srcopsmetrics/storage.py

```diff
@@ -138,19 +140,23 @@ def load_previous_knowledge(
         """Load previously collected repo knowledge. If a repo was not inspected before, create its directory.
 
         Arguments:
-            file_path {Optional[Path]} -- path to where was stored previous knowledge from
-                               inspected github repository
+            file_path {Optional[Path]} -- path to previously stored knowledge from
+                               inspected github repository. If None is passed, the used path will
+                               be the
```

Comment:
the used path will be the what?

Reply:
@KPostOffice hope it's correctly linked now?